### ISSUE TITLE
Ctrl / Cmd + Click no longer opens project in vendor directory

### ISSUE THREAD

**Issue by andyexeter** [psliwa/idea-composer-plugin#75]
When I Ctrl + Click on a package in the composer.json file it no longer navigates to that package within the vendor directory. Instead it now opens https://schemastore.azurewebsites.net/schemas/json/composer.json as a new tab in the editor.

I initially thought this functionality was native to PhpStorm but after creating [an issue](https://youtrack.jetbrains.com/issue/WI-44731) there I was informed that the functionality is provided by this plugin.

I'm not sure when this started happening, but I was using PhpStorm 2018.3.2 when I reported it and it had been happening for a few releases before that. It is still happening using PhpStorm 2018.3.4 and v1.0.26 of this plugin.

Are you able to fix this? It was a really useful feature. Below is a gif showing what currently happens when using Ctrl + Click on a package:

![ezgif-5-f49623e9b10b](https://user-images.githubusercontent.com/6660584/52593221-4e6c0f00-2e40-11e9-9770-fe38c4c949fb.gif)

**Comment by speto** [comment]
Same here. Navigation feature hasn't been working for last months. Now I'm on PhpStorm 2018.3 and plugin v1.0.26. Behaviour is same as @andyexeter described.

**Comment by johnpancoast** [comment]
This lost/broken behavior was very important to traverse and learn dependencies directly in the IDE. Any word from the maintainer(s)?

**Comment by psliwa** [comment]
Thanks for reporting the issue @andyexeter and @speto. After my quick investigation I think this is not the plugin issue. PhpStorm introduced built-in json schema features, eg. to composer.json file and right now cmd+click on json key opens json-schema definition as far as I can see. It looks like new PhpStorm features hide some features of this plugin - soon I will look into it if I can do something with that.

@johnpancoast I cannot effort to spend the same amount of time for this plugin as I used to, so please skip putting me on the spot like this. Please have in mind a lot of open source projects are voluntary work for the community exactly the same as this plugin. So you are not a client who can demand something because you paid for the product, you are the part of this community and you can contribute in various ways - eg. by raising a ticket as @andyexeter did, confirming the issue as @speto did, provide more information, prepare PR, fix typos in docs or code, propose new features etc. Everyting with culture and respect to the others.

**Comment by johnpancoast** [comment]
@psliwa, whoa, I think you completely misunderstood my post to be more negative than it was. I was simply stating that this is an important feature and that, as @speto pointed out, this functionality has been broken for months. So I was asking if there was a status update from the maintainers _which is **far** from unordinary_. Just because I'm asking for a status update on an important broken feature does not mean that I misunderstand how open source works or that I need the lesson on it. Quite the contrary.

It's fair for the community to ask for a status update, especially _because_ of the fact that this is an open source project. For example, perhaps you've started on fixing this and someone else can pick up where you left off or perhaps it's something where the community needs to start  for themselves and you can offer pointers.

I've worked in the field long enough to know how open source works and don't need the lesson or the attitude, and I've asked the same question of other important issues on other libraries/repos and gotten polite responses on the status of the issue or been pointed to where to help out. Perhaps you took my post a little personally, but that was not my intent. I'm just trying to find out the status and will possibly look into this myself.

**Comment by psliwa** [comment]
@johnpancoast probably I misinterpreted your previous comment - in Polish such sentence has rather negative sense, but in English probably not. In written communication a lot of meaning is lost, because tone or emotions are hard to express just writting, so reader may get something diffrently than writter's intentions. I apologize for that.

**Comment by johnpancoast** [comment]
Hi @psliwa, I realized it may have been due to language differences too, and also the fact that text communication can be misunderstood, like you said. I was probably a too short and blunt in my message and I can see how it could have been read negatively. It was a simple misunderstanding which was partially my fault. No hard feelings. All is good.

**Comment by psliwa** [comment]
I checked the issue in more details. I did an experiment, I configured plugin in order to work with different file name than `composer.json` (eg. `composer-bak.json`) and in such case this feature works correctly, so the plugin logic works ok. I think PhpStorm new json-schema and composer features hide this particular feature. I will share with my findings in the PhpStorm issue what @andyexeter created: https://youtrack.jetbrains.com/issue/WI-44731
